controllers/feeds/getFeedsV2.js

```diff
@@ -117,14 +116,17 @@ module.exports = async (req, res) => {
       listBlockUser,
       listBlockDomain,
       listPostAnonymousAuthor,
-      isBlurredPost
+      isBlurredPost,
+      allFollowingUser
     ] = await Promise.all([
       UsersFunction.findAnonymousUserId(User, req.userId, {raw: true}),
       getListBlockUser(req.userId),
       getBlockDomain(req.userId),
       getListBlockPostAnonymousAuthor(req.userId),
-      UserFollowUserFunction.checkIsBlurredPost(UserFollowUser, req.userId)
+      UserFollowUserFunction.checkIsBlurredPost(UserFollowUser, req.userId),
+      UserFollowUserFunction.getAllFollowingUser(UserFollowUser, req.userId)
     ]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `allFollowingUser` in the Promise.all array and its usage later in the code seems fine. However, ensure that the function `UserFollowUserFunction.getAllFollowingUser` handles any potential errors and does not cause the entire Promise.all to fail.



<!-- This is an auto-generated comment by OSS CodeRabbit -->